### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jaamanu**
Created tables through rails migration
I wrote models for the app
I queried the database via controllers and populated the data to the frontend via the view files
I set up the project with PostgreSQL database
I followed the design guidelines provided
I made sure I met up with project requirements
I tested my models, controllers, and also wrote integration tests for the app

https://www.loom.com/share/dd87d59d78ac4a46a8bd18714e2a8319

### CODE REVIEW

File: app/views/groups/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                    <p><%= "$#{number_with_delimiter(group.entity.amount, :delimiter => ',')}" %></p>
+                </div>
+            </div>
+        </div>
+    <% end %>
+    <% end %>
+
+    <%= button_to 'Back', root_path, class:'button', method: :get %>
+</div>
+    <%= link_to 'Add new transaction', new_group_entity_path(@group), class:'fixIt' %>
```
Comment (OLIPLICHE) [inline app/views/groups/show.html.erb:30]: 👍  App working fine
Capture

File: README.md [hunk 2]
```diff
@@ -283,6 +261,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (OLIPLICHE) [inline README.md:262]: - [ ] Hi, I would advise you to link MIT to the license. Tho it is part of the project requirements, it makes your project more professional and be trusted

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @jaamanu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Highlights
- App is working fine 👍 
- Unit tests passed 👍 
- Integration tests 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (jaamanu) [reply README.md:262]: @ olipliche 
I am not impress with your review, you took over 1 hour and yet disapprove a project which has no errors, the required changes as you mentioned is not part of the project requirement.

Comment (OLIPLICHE) [reply README.md:262]: Hi @jaamanu, Thank you for reaching out. I mentioned in my comment that it is part of the general requirements. The general requirements belong to your project. We request this not block the students. Your journey is not only about coding but also about teaching you to become more professional. Sorry for any disagreements. All the best

**Review by DeliceLydia — APPROVED** [review]
## Hi @jaamanu, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


### Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**